apps/ejabberd/src/mongoose_api_client.erl

```diff
@@ -0,0 +1,145 @@
+%%%-------------------------------------------------------------------
+%%% @author ludwikbukowski
+%%% @copyright (C) 2016, <COMPANY>
```

Comment:
Please fill in <COMPANY> or remove the section completely as we can read this info from git.